src/domain/events/index.ts

```diff
@@ -21,6 +22,9 @@ import { ConditionalOrder, ConditionalOrderParams } from "@cowprotocol/cow-sdk";
 
 import { ChainContext } from "../../services/chain";
 
+const composableCow = ComposableCoW__factory.createInterface();
+const log = getLogger("addContract:_addContract");
```

Comment:
Why is this log saying `addContract` if its just on top of this file which is called `events`?

Reply:
I don't know, I just moved this code from the `_addContract` function in order to not create excessive instances

Reply:
I thought so. In principle are instances that are scoped to that function, so the garbage collector should be able to find (unless we have a leak in the logging library)

If you really want to do this, at least we need to find a better name than just `log`. I might want to add another function `doSomething()` and use that log, and I will see `addContract` in my log messages

Reply:
Got you, reverted the logger back and kept only `composableCow`